helm-frontend/src/components/LeaderboardTables.tsx

```diff
@@ -43,11 +43,21 @@ export default function LeaderboardTables({
     value: string;
   }
 
+  function truncateHeader(value: string): string {
+    if (value.length > 30) {
+      return value.substring(0, 27) + "...";
```

Comment:
Can also consider using `text-overflow: ellipses` ([doc](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow))

Reply:
Tried this and for whatever reason the overflow wasn't triggering, but I'm sure its resolvable so I left this as a TODO.